cmd/tempo/build/build.go

```diff
@@ -5,26 +5,6 @@ import (
 	prom "github.com/prometheus/prometheus/web/api/v1"
 )
 
-// Version information passed to Prometheus version package.
-// Package path as used by linker changes based on vendoring being used or not,
-// so it's easier just to use stable Tempo path, and pass it to
-// Prometheus in the code.
-var (
-	Version   string
-	Revision  string
-	Branch    string
-	BuildUser string
-	BuildDate string
-)
-
-func init() {
-	version.Version = Version
-	version.Revision = Revision
-	version.Branch = Branch
-	version.BuildUser = BuildUser
-	version.BuildDate = BuildDate
-}
-
 func GetVersion() prom.PrometheusVersion {
```

Comment:
There is only one call which is in the `/status/version` handler.  This could also be replaced with the `version.Info()` call we see in `main()` and then this entire package could get deleted.  Thoughts?